src/fidesops/ops/schemas/connection_configuration/connection_secrets.py

```diff
@@ -45,10 +45,24 @@ class Config:
 
         extra = Extra.forbid
         orm_mode = True
+        schema_extra = {
+            "example": {
+                "url": "http://wiseai.dev/"
```

Comment:
Please avoid using real world URLs in example data, preference `https://example.com`